### ISSUE TITLE
scroll problem on 0.4.0-dev+1 

### ISSUE THREAD

**Issue by wonpyohong** [bnxm/implicitly_animated_reorderable_list#52]
reproducible example.

```
import 'dart:ui';

import 'package:flutter/material.dart';
import 'package:implicitly_animated_reorderable_list/implicitly_animated_reorderable_list.dart';
import 'package:implicitly_animated_reorderable_list/transitions.dart';

Future<void> main() async {
  WidgetsFlutterBinding.ensureInitialized();
  print('main');

  runApp(
    MaterialApp(
      home: MyApp(),
    ),
  );
}

class MyApp extends StatefulWidget {
  @override
  _MyAppState createState() => _MyAppState();
}

class _MyAppState extends State<MyApp> with TickerProviderStateMixin {

  final items = [5555555555551, 5555555555552, 5555555555553, 5555555555554, 5555555555555,
    5555555555556, 5555555555557, 5555555555558, 5555555555559, 55555555555510,
    55555555555511, 55555555555512, 55555555555513, 55555555555514, 55555555555515,
    55555555555516, 55555555555516];

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: null,
      body: ImplicitlyAnimatedReorderableList<int>(
        items: items,
        areItemsTheSame: (oldItem, newItem) => oldItem == newItem,
        onReorderFinished: (item, from, to, newItems) {
          // Remember to update the underlying data when the list has been
          // reordered.
          setState(() {
            items
              ..clear()
              ..addAll(newItems);
          });
        },
        itemBuilder: (context, itemAnimation, item, index) {
          // Each item must be wrapped in a Reorderable widget.
          return Reorderable(
            // Each item must have an unique key.
            key: ValueKey(item),
            // The animation of the Reorderable builder can be used to
            // change to appearance of the item between dragged and normal
            // state. For example to add elevation when the item is being dragged.
            // This is not to be confused with the animation of the itemBuilder.
            // Implicit animations (like AnimatedContainer) are sadly not yet supported.
            builder: (context, dragAnimation, inDrag) {
              final t = dragAnimation.value;
              final elevation = lerpDouble(0, 8, t);
              final color = Color.lerp(Colors.white, Colors.white.withOpacity(0.8), t);

              return SizeFadeTransition(
                sizeFraction: 0.7,
                curve: Curves.easeInOut,
                animation: itemAnimation,
                child: Material(
                  color: color,
                  elevation: elevation,
                  type: MaterialType.transparency,
                  child: Handle(
                    delay: const Duration(milliseconds: 1000),
                    child: ListTile(
                      title: Text(item.toString()),
                      // The child of a Handle can initialize a drag/reorder.
                      // This could for example be an Icon or the whole item itself. You can
                      // use the delay parameter to specify the duration for how long a pointer
                      // must press the child, until it can be dragged.
                    ),
                  ),
                ),
              );
            },
          );
        },
      ),
    );
  }
}
```

This code is similar to readme example on pub.dev page.
difference: I lifted Handle up to ListTile

This code scrolls well on 0.3.2, but not on 0.4.0-dev+1

**Comment by bnxm** [comment]
I think I was able to fix in it version 0.4.0-dev+2. You might wanna try and see if it works now.

**Comment by wonpyohong** [comment]
thanks! but i found another problem.. I will make a new issue
